src/components/shared/StepWidget.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { TiTick } from 'react-icons/ti';
+import '../../assets/styles/components/stepper.scss';
+
+type stepperProps = { steps: string[] };
+
+function Stepper({ steps }: stepperProps) {
+  const [currentStep, setCurrentStep] = useState(1);
+  const [complete, setComplet] = useState(false);
+
+  function next() {
+    if (currentStep === steps.length) setComplet(true);
+    else {
+      setCurrentStep((prev: number) => prev + 1);
+    }
+  }
+
+  function previous() {
+    if (currentStep > 1) {
+      setComplet(false);
+      setCurrentStep((prev: number) => prev - 1);
+    }
+  }
+
+  return (
```

Comment:
This looks great, however the small design can do with some improvements. If you can maybe try to reduce the space between these ```steps``` cause in small devices the component is taking on a huge screen height.. reduce the size of everything to make it look appropriate for the small screen